src/commands/git/worktree.ts

```diff
@@ -110,6 +110,7 @@ interface DeleteState {
 	repo: string | Repository;
 	uris: Uri[];
 	flags: DeleteFlags[];
+	doNotSuggestDeletingBranches?: boolean;
```

Comment:
Maybe we can just pass `branch` or `branches` as state when coming from "delete branch" and then we don't need to calculate them on the delete steps, and also can use their existence in state as a flag to restrict the options?

Reply:
@axosoft-ramint 

What do you think about this:
<img width="673" alt="image" src="https://github.com/user-attachments/assets/33ac5b59-16a5-4580-8530-d3335498433f">

Reply:
@axosoft-ramint 

>  then we don't need to calculate them on the delete steps

It's not clear how we can use the passed branches on the "worktrees" step. We calculate branches only if "Delete Worktrees&Branches" is selected, but those are hidden.

Reply:
Fair enough - if the state isn't needed then a boolean should suffice 👍🏼

Reply:
@axosoft-ramint 

I'm thinking about renaming because it affects the view in a very specific case. "Deleting of the worktree of a selected branch". So, it can be called like this: `deletingOfSelectedBranches`, or moreover I'm thinking of replacing the field by a flag:

```
type DeleteFlags = '--force' | '--delete-branches' | '--deleting-of-selected-branches';
```

What do you think?

Reply:
@axosoft-ramint 

I've just pushed how it can be with a `--deleting-of-selected-branches` flag. Let me know if you prefers it being a boolean field.

Reply:
The UI:

<img width="732" alt="image" src="https://github.com/user-attachments/assets/d78a30f8-7631-4ac8-9d8c-2bb55a253cc3">

<img width="661" alt="image" src="https://github.com/user-attachments/assets/823dd5ad-35e2-4675-ac78-e2e80f06989f">

Unfortunately, I need to go now. I can do any other updates tomorrow.

Reply:
@sergeibbb No preference here. Up to you. Doing it with a flag like that sounds fine. My main hope is that the messaging on the worktree step allows me to understand that we are deleting the worktree because it is necessary to delete the branch.

Reply:
@axosoft-ramint 

Ok. I'm leaving it with the flag. Feel free to suggest updates for the wording, so it would be more clearer for user. You are the native speaker, so, you know it better.

Reply:
@sergeibbb I think we can take out the word "corresponding" in the options since it becomes a bit wordy. We should also make sure the capitalization is consistent in the options.

What do you think about "Delete Worktree of Branch"/"Delete Worktrees of Branch" etc.?

Reply:
@sergeibbb I updated the wording on your branch. Thanks for working on this!

Reply:
Great! Thank you!